### DISCUSSION TITLE
Is there a keymap for `trigger inline suggestion`

### DISCUSSION THREAD

**Discussion by ChenZhongPu** [zbirenbaum/copilot.lua#147] [Q&A] [answered]
In VSCode, `copilot` supports the keymap `<M>+\` for `trigger inline suggestion`.  Is there a corresponding keymap in `copilot.lua`?

**Comment by ChenZhongPu** [comment]
I tried to manually trigger it by `<M+]>` or `<M+[`, where `M` is Option in MacOS. It seems a workaround way. But in MacOS, those two keys will also mapped to special characters (i.e., `'` and `"`), and such mappings are very annoying.

**Reply by MunifTanjim** [reply]
Using `next` or `prev` keymap is the recommended way:

https://github.com/zbirenbaum/copilot.lua/blob/f0b41fb89ef2ed7ab4e8cd5208a30755aa638fc4/README.md?plain=1#L112-L114

You can change the default keymaps if `<M-]>` or `<M+[>` does not work on your OS and/or terminal.

**Comment by ChenZhongPu** [comment]
It may caused by my terminal settings.  Which terminal do you recommend? I also tried both Konsole in Manjaro and iterm2 in MacOS, but `<M>` key did not work as expected.

**Reply by MunifTanjim** [reply, marked as answer]
I've used Alacritty and Kitty on both Linux and MacOS. Both of them works alright.

For Alacritty on MacOS you'll need these settings if you want the terminal to see `Alt` when you press `Option`: https://github.com/MunifTanjim/dotfiles/blob/6b5199346f7e96065d5e517e61e2d8768e10770d/private_dot_config/alacritty/darwin.yml#L7-L90

For Kitty, I don't remember doing something specific. But here's my config just in case: https://github.com/MunifTanjim/dotfiles/blob/6b5199346f7e96065d5e517e61e2d8768e10770d/private_dot_config/kitty/kitty.conf.tmpl

If `<M-` still doesn't work for you, you'll need to use some other keybinding.
